src/context/offline.rs

```diff
@@ -98,10 +98,44 @@ impl OfflineAudioContext {
         }
     }
 
+    /// Given the current connections and scheduled changes, starts rendering audio.
+    ///
+    /// Rendering an OfflineAudioContext is CPU-bound (involves no IO, and no `.await` points) so
```

Comment:
We should probably add a call to https://docs.rs/tokio/latest/tokio/task/fn.yield_now.html for `N` rendered samples. `N` yet to be determined